bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -47,7 +47,7 @@ spec:
         > 0.1
       for: 15m
       labels:
-        severity: warning
+        severity: info
```

Comment:
same for `OVNKubernetesNodePodDeleteError`

Reply:
done